sdk/typescript/README.md

```diff
@@ -32,16 +32,32 @@ Fetch transaction details from a transaction digest:
 
 ```typescript
 import { JsonRpcProvider } from '@mysten/sui.js';
-const provider = new JsonRpcProvider('http://127.0.0.1:5001/');
+const provider = new JsonRpcProvider('https://gateway.devnet.sui.io:9000/');
 const txn = await provider.getTransaction(
   '6mn5W1CczLwitHCO9OIUbqirNrQ0cuKdyxaNe16SAME='
 );
 ```
 
 For any operations that involves signing or submitting transactions, you should use the `Signer` API. For example:
 
-To sign a raw message:
-TODO
-
 To transfer a Coin<SUI>:
+
+```typescript
+import { Ed25519Keypair, JsonRpcProvider, RawSigner } from '@mysten/sui.js';
+// Generate a new Keypair
+const keypair = new Ed25519Keypair();
+const signer = new RawSigner(
+  keypair,
+  new JsonRpcProvider('https://gateway.devnet.sui.io:9000/')
+);
+const txn = await signer.transferCoin({
+  signer: keypair.getPublicKey().toSuiAddress(),
+  objectId: '5015b016ab570df14c87649eda918e09e5cc61e0',
```

Comment:
Should this now have an `0x` prefix per?
https://github.com/MystenLabs/sui/pull/1867